### PR TITLE
CP-1829/refactor documents.

### PR DESCRIPTION
**Pull request by Azharhussain20**
Updated the documents with the refactor methods of tags and added story methods.

### CODE REVIEW

File: docs/sdks/ios/methods.md [hunk 1]
```diff
@@ -47,11 +47,36 @@ let isSubscribed = CleverPush.isSubscribed()
 Objective-C:
 
 ```objective-c
+// get available tags
 NSArray* channelTags = [CleverPush getAvailableTags];
 
-[CleverPush addSubscriptionTag:@"TAG_ID"];
+// add/remove tag with action callback
+[CleverPush addSubscriptionTag:@"TAG_ID" callback:^(NSString *tagId) {
```
Comment (mariusglauer) [inline docs/sdks/ios/methods.md:54]: Nice, can you also leave the "old" method without callback in the docs? So the user knows both ways are possible.

File: docs/sdks/ios/methods.md [hunk 2]
```diff
@@ -63,11 +88,36 @@ NSArray* subscriptionTopics = [CleverPush getSubscriptionTopics];
 Swift:
 
 ```swift
+// get available tags
 let channelTags = CleverPush.getAvailableTags()
 
-CleverPush.addSubscriptionTag("TAG_ID")
+// add/remove tag with action callback
+CleverPush.addSubscriptionTag("TAG_ID", callback: { tagId in
```
Comment (mariusglauer) [inline docs/sdks/ios/methods.md:95]: Nice, can you also leave the "old" method without callback in the docs? So the user knows both ways are possible.

File: docs/sdks/ios/methods.md [hunk 3]
```diff
@@ -63,11 +88,36 @@ NSArray* subscriptionTopics = [CleverPush getSubscriptionTopics];
...
+CleverPush.removeSubscriptionTags(tags, callBack: { remainingTags in
+    print(remainingTags as Any)
+})
+
+// add/remove tag without action callback
+CleverPush.addSubscriptionTag("TAG_ID", callback: nil)
+CleverPush.removeSubscriptionTag("TAG_ID", callback: nil)
+
+// add/remove multiple tags without action callback
+CleverPush.addSubscriptionTags(tags, callBack: nil)
```
Comment (mariusglauer) [inline docs/sdks/ios/methods.md]: `callBack` instead of `callback` ?

File: docs/sdks/ios/methods.md [hunk 4]
```diff
@@ -288,3 +338,37 @@ import CleverPush
 let chatView = CPChatView(frame: CGRect(x: 0, y: 0, width: self.view.frame.size.width, height: self.view.frame.size.height))
 self.view.addSubview(chatView)
 ```
+
+## Story
```
Comment (mariusglauer) [inline docs/sdks/ios/methods.md]: Please rename to "Stories" and add 1-2 sentences of description what stories are and add 1-2 screenshots.

File: docs/sdks/ios/methods.md [hunk 5]
```diff
@@ -288,3 +338,37 @@ import CleverPush
...
+                            backgroundColor:[UIColor greenColor]
+                            textColor:[UIColor blackColor]
+                            fontFamily:@"AppleSDGothicNeo-Bold"
+                            borderColor:[UIColor redColor]
+                            storyWidgetId:@"STORY_WIDGET_ID"];
+
+[self.view addSubview:storyView];
+```
+Swift:
+1. import CleverPush
```
Comment (mariusglauer) [inline docs/sdks/ios/methods.md]: import -> Import

**Review by mariusglauer — CHANGES_REQUESTED** [review]

File: docs/sdks/ios/methods.md (hunk 2 above)
Comment (Azharhussain20) [reply docs/sdks/ios/methods.md:95]: @gbhrdt I have already added without callback methods.

File: docs/sdks/ios/methods.md [hunk 6]
```diff
@@ -47,11 +47,36 @@ let isSubscribed = CleverPush.isSubscribed()
...
+    NSLog(@"%@",remainingTags);
+}];
 
-[CleverPush removeSubscriptionTag:@"TAG_ID"];
+// add/remove tag without action callback
+[CleverPush addSubscriptionTag:@"TAG_ID" callback:nil];
+[CleverPush removeSubscriptionTag:@"TAG_ID" callback:nil];
+
+// add/remove multiple tags without action callback
+[CleverPush addSubscriptionTags:tags callBack:nil];
```
Comment (mariusglauer) [inline docs/sdks/ios/methods.md]: callBack -> callback

**Review by mariusglauer — CHANGES_REQUESTED** [review]

**Pull request merged**